libs/psqlodbc/Makefile

```diff
@@ -5,17 +5,20 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=psqlodbc
-PKG_VERSION:=11.00.0000
-PKG_RELEASE:=2
+PKG_VERSION:=17.00.0004
+PKG_RELEASE:=1
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://ftp.postgresql.org/pub/odbc/versions/src
-PKG_HASH:=703e6b87022f95ffa00d9f86c8f0a877f8a55b9b3be0942081f382e794112a86
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/postgresql-interfaces/psqlodbc.git
+PKG_SOURCE_DATE:=2024-12-09
+PKG_SOURCE_VERSION:=20097cdf913de593b17d8aa9ec163de4e12896c0
```

Comment:
Can't we use github codeload and download the tarball for the [REL-17_00_0004](https://github.com/postgresql-interfaces/psqlodbc/releases/tag/REL-17_00_0004) tag instead of a complete git checkout and locally created mirror tarball?

Reply:
It's just my preference not to rely on GH and their generated tarballs, I dont like relying on tags in general